content/en/docs/release_notes/3_0_1_20211018_relnotes.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Axway CLI 3.0.1 GA release notes - 18 October 2021
+linkTitle: Axway CLI 3.0.1 GA release notes - 18 October 2021
+description: Review Axway CLI release details such as improvements and bug fixes.
+weight: 10
+date: 2021-10-18
+---
+
+{{% variables/axway_cli_prod_name %}} 3.0.1 GA - 18 October 2021 is a patch release that includes bug fixes.
+
+## Installation
+
+```
+npm i -g axway
+```
+
+## Improvements
```

Comment:
The description says bug fix but the heading says Improvements. Should the heading be changed Bug Fix (or Breaking Change; not sure if that is the same or different)?

Reply:
Fixed